pipelines/build/common/openjdk_build_pipeline.groovy

```diff
@@ -437,6 +437,36 @@ class Build {
         return fileName
     }
 
+    def buildScripts(context, cleanWorkspace, buildConfig, filename) {
```

Comment:
Can this function be split up?  It seems to do two disparate things.

Reply:
@karianna I didn't want to modify the function as it's existing code, happy to split up later but that should be a separate PR